File: Java/백준/Bronze/2908. 상수/상수.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        String a = st.nextToken();
        String b = st.nextToken();

        int a_rev = Integer.parseInt(new StringBuilder(a).reverse().toString());
        int b_rev = Integer.parseInt(new StringBuilder(b).reverse().toString());

        System.out.println(Math.max(a_rev, b_rev));
    }
}

File: C99/백준/Bronze/2908. 상수/상수.c
#include <stdio.h>

int reverse(int n) {
    int rev = 0;
    while (n > 0) {
        rev = rev * 10 + n % 10;
        n /= 10;
    }
    return rev;
}

int main() {
    int a, b;
    scanf("%d %d", &a, &b);

    int a_rev = reverse(a);
    int b_rev = reverse(b);

    if (a_rev > b_rev) {
        printf("%d\n", a_rev);
    } else {
        printf("%d\n", b_rev);
    }

    return 0;
}
